intranet/apps/users/models.py

```diff
@@ -134,14 +134,41 @@ def get_teachers_attendance_users_sorted(self) -> "QuerySet[User]":  # noqa
         """Returns a ``QuerySet`` containing both teachers and attendance-only users sorted by
         last name, then first name.
 
-        This is used for the announcement request page.
-
         Returns:
             A ``QuerySet`` of teachers sorted by last name, then first name.
 
         """
         return self.get_teachers_attendance_users().order_by("last_name", "first_name")
 
+    def get_approve_announcements_users(self) -> "QuerySet[User]":  # noqa
+        """Returns a ``QuerySet`` containing all users except simple users, tjstar presenters,
+        alumni, service users and students.
+
+        Returns:
+            A ``QuerySet`` of all users except simple users, tjstar presenters, alumni,
+            service users and students.
+
+        """
+
+        users = User.objects.all().exclude(user_type__in=["simple_user", "tjstar_presenter", "alum", "service", "student"])
```

Comment:
I would recommend making this a `filter` rather than `exclude` (in case, future developers want to add additional user types).

Also, `.all()` is not necessary when dealing with `User.objects`.

Reply:
Fixed?

Reply:
Yes.